cool_oneliners/xml_search_schema.nu

```diff
@@ -0,0 +1,6 @@
+# Search the WordprocessingML XML Schema definition file for a simple type by name
+# You'll need the wml.xsd file.
+# To get that file, first download the following zip:
```

Comment:
The fact I needed to traverse 2 zip archives for the XML file I need has me quietly chuckling.